content/spin/v2/http-outbound.md

```diff
@@ -52,31 +52,64 @@ Generally, you should use `IncomingResponse` when you need to stream the respons
 Here is an example of doing outbound HTTP in a simple request-response style:
 
 ```rust
-use spin_sdk::http::{IntoResponse, Request, send};
-use spin_sdk::http_component;
+use spin_sdk::{
+    http::{IntoResponse, Request, Method, Response},
+    http_component,
+};
 
 #[http_component]
 // The trigger handler (in this case an HTTP handler) has to be async
 // so we can `await` the outbound send.
 async fn handle_request(_req: Request) -> anyhow::Result<impl IntoResponse> {
 
-    // For this example, use the spin_sdk::http::RequestBuilder type
-    // for the outbound request.
-    let outbound_req = Request::get("https://www.fermyon.com/");
+    // Create the outbound request object
+    let request = Request::builder()
+        .method(Method::Get)
+        .uri("https://www.fermyon.com/")
+        .build();
 
-    // Send the outbound request, capturing the response as raw bytes
-    let response: http::Response<Vec<u8>> = send(outbound_req).await?;
+    // Send the request and await the response
+    let response: Response = spin_sdk::http::send(request).await?;
 
     // Use the outbound response body
     let response_len = response.body().len();
 
-    Ok(http::Response::builder()
+    // Return the response to the inbound request
+    Ok(Response::builder()
         .status(200)
         .header("content-type", "text/plain")
-        .body(format!("The test page was {response_len} bytes"))?)
+        .body(format!("The test page was {response_len} bytes"))
+        .build())
 }
 ```
 
+The outbound request will only work if we [grant network permissions to the component](https://developer.fermyon.com/spin/v2/redis-outbound#granting-network-permissions-to-components). Please update the `spin.toml` file at the component level as follows:
+
+```toml
+[component.example]
+allowed_outbound_hosts = ["https://www.fermyon.com:443"]
+```
+
+We can now build and run the application:
```

Comment:
This feels like tutorial content.  This page seeks to act as a reference page for the API - I'm not keen on repeating "how to build, run, and invoke a Spin app" just because it contains a fragment showing that API.  Maybe move the "now build and run" stuff out of the API guide and into a tutorial that you link from here?